bin/zen_restart_runzope

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# zen_restart_runzope  shell script restarts runzope when its memory leaks too much.
+#
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2007, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+# set the max to 2G
+(( max_m=2*1024*1024 ))
+
+# 10 minutes
+poll_t=600
```

Comment:
It would be good if we could find a way to allow overrides from the CC for `max_m` and `poll_t`.